src/internal/module/list.go

```diff
@@ -87,7 +89,7 @@ func (modules ModuleList) Parallel(maxConcurrency int, action Action) error {
 
 	errs := parallel.ForEach(actions, maxConcurrency)
 	if len(errs) != 0 {
-		return fmt.Errorf("Encountered %d errors processing %d modules", len(errs), len(modules))
+		return fmt.Errorf("encountered %d errors processing %d l", len(errs), len(l))
```

Comment:
Looks like a find/replace problem in this message.

Reply:
Good catch, I'll fix this.